### PR TITLE
move genesis.blob to gcs bucket

### PR DESCRIPTION
**Pull request by aluon**
## Description

Move genesis.blob from a Secret to GCS

## Type of Change
- [x] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [x] Developer Infrastructure
- [ ] Other (specify)

## How Has This Been Tested?

See the Forge run for this PR

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by aluon** [comment]
* **#12474** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-core/12474?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @aluon and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: terraform/helm/genesis/files/genesis.sh [hunk 1]
```diff
@@ -129,12 +129,18 @@ kubectl get pvc -o name | grep /fn- | grep -v "e${ERA}-" | xargs -r kubectl dele
 # delete all genesis secrets except for those from this era
 kubectl get secret -o name | grep "genesis-e" | grep -v "e${ERA}-" | xargs -r kubectl delete
 
+# Upload genesis.blob to GCS
```
Comment (ibalajiarun) [inline terraform/helm/genesis/files/genesis.sh:132]: We can also gzip the blob. May make uploading faster.

**Review by ibalajiarun — COMMENTED** [review]
Can we also measure the time it takes to upload a 20MB blob and make sure it's relatively fast?

**Comment by aluon** [comment]
I created a PR to sync these changes using copybara instead https://github.com/aptos-labs/aptos-core/pull/12524

**Pull request closed without merging**